src/admin/redux/modules/letter.js

```diff
@@ -36,8 +36,12 @@ const getAddress = (
   , postcode
   }
 ) =>
-  [ `${title} ${first_name[0] || ''} ${last_name}`, address1, address2,
-    address3, address4, county, postcode ]
+  [ `${title} ${(first_name && first_name[0]) || ''} ${last_name}`
```

Comment:
Yikes! Property access in JS :(

BTW, solving this with some of the FP libraries might look like:

```js
// Sanctuary
S.compose(S.maybe('', S.I), S.head)

// Ramda
R.propOr('', 0)
```